.github/workflows/test.yml

```
@@ -0,0 +1,97 @@
+
+    - name: Install dependencies
+      working-directory: ./websocket-server
+      run: npm ci
+
+    - name: Run tests
+      working-directory: ./websocket-server
+      run: npm run test:ci
+
+    - name: Upload coverage reports
+      if: matrix.node-version == '20.x'
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)